config/metrics/crd-metrics-config.yaml

```diff
@@ -0,0 +1,676 @@
+---
+kind: CustomResourceStateMetrics
+spec:
+  resources:
+  - errorLogV: 0
+    groupVersionKind:
+      group: infrastructure.cluster.x-k8s.io
+      kind: VSphereCluster
+      version: v1beta1
+    labelsFromPath:
+      cluster_name:
+      - metadata
+      - labels
+      - cluster.x-k8s.io/cluster-name
+      name:
+      - metadata
+      - name
+      namespace:
+      - metadata
+      - namespace
+      uid:
+      - metadata
+      - uid
+    metricNamePrefix: capv_vspherecluster
```

Comment:
@sbueringer : wdyt: Use `capv_` or `capi_` as prefix?

When having `capv_` as prefix: is there a potential of overlap of names in-between providers?

Reply:
Let's go with capi. As we always add the CRD name after we would also have a CRD name conflict if two providers have a CRD with the same name. That seems to be okay